.circleci/config.yml

```diff
@@ -209,6 +209,51 @@ jobs:
             docker login -u "$DOCKER_USER" -p "$DOCKER_PASS"
             make -C ./docker version=${CIRCLE_TAG} alias=latest push
 
+  publish-docker-image-arm64:
+    machine:
+      image: ubuntu-2004:current
+    resource_class: arm.medium
+    steps:
+      - checkout
+      - run:
+          name: check if emscripten-releases-tags.json has changed
+          command: |
+            CHANGED_FILES=$(git diff --name-only <<pipeline.git.base_revision>>..<<pipeline.git.revision>>)
+            if [[ ! $CHANGED_FILES =~ "emscripten-releases-tags.json" ]]; then
+              echo "emscripten-releases-tags.json has not changed, skipping build and publish"
+              exit 0
+            fi
+      - run:
+          name: extract latest emscripten arm64 linux version
+          command: |
+            export ARM64_VERSION=$(grep -oP '(?<="latest-arm64-linux": ")[^"]*' emscripten-releases-tags.json)
+            export ARM64_IMAGE_TAG=${ARM64_VERSION}-arm64
```

Comment:
To these exported variables persist to the next `run` ?  My recollection was that they do not, and that you need to persist them somehow/somewhere?

Reply:
You are correct! Thanks for pointing that out.

I've simulated it on my own CircleCI account to ensure that this job works as expected.

You can see the steps running at https://app.circleci.com/pipelines/github/jamsinclair/emsdk-circleci-test/8/workflows/8dffc43c-ef41-43b5-a973-c8778255612d/jobs/8